modules/veteran/app/workers/veteran/vso_reloader.rb

```diff
@@ -3,54 +3,58 @@
 require 'sidekiq'
 
 module Veteran
-  class VsoReloader
-    include Sidekiq::Worker
-    BASE_URL = 'https://www.va.gov/ogc/apps/accreditation/'
-
+  class VsoReloader < BaseReloader
     def perform
       array_of_organizations = reload_representatives
-      Veteran::Service::Organization.import(array_of_organizations, on_duplicate_key_ignore: true)
+      Veteran::Service::Representative.where.not(representative_id: array_of_organizations).destroy_all
```

Comment:
this is a rare case for me where I think a comment would be helpful. is this a de-duping operation? what is intended to be destroyed?

Reply:
I agree a comment would help context. I believe the reason is we are not informed of removals to the list so we clean the slate and start over each time

Reply:
So the assumption is made early on that every time the list is updated, if anyone has been removed from the list that may mean they are no longer an authorized VSO from OGC so the decision was made to remove them from our list accordingly. If they make it back on the list in the future that's fine but since we are using OGC as the source of truth here then it makes the most sense to remove anyone not in the incoming list. I'll add a comment to address this question.

Reply:
Comment added.